one/src/lib.rs

```diff
@@ -150,6 +142,16 @@ struct DaemonOpts {
     )]
     idle_conns_timeout_ms: u64,
 
+    #[command(flatten)]
+    db_opts: DBOpts,
+}
+
+#[derive(Args, Debug)]
+struct DBOpts {
+    /// Path to storage directory
+    #[arg(short, long, env = "CERAMIC_ONE_STORE_DIR")]
+    store_dir: Option<PathBuf>,
```

Comment:
doesn't this have a default value?  I don't think it need to be specified currently for the daemon to start up - why isn't the default specified here?

Reply:
The default is system dependent as its the user's home directory which changes based on the user and the OS. Also note this is not a change in behavior but a refactor to enable the migrations cmd to share this default logic.